guide/yaml/blueprinting-tips.md

```diff
@@ -54,26 +69,64 @@ container (i.e. no install artifacts from previous runs), while taking advantage
 of starting containers.
```

Comment:
There is a link to `[Clocker project](http://www.clocker.io)` a few lines higher up.